README.md

```diff
@@ -23,25 +23,32 @@ Cataclysm: Dark Days Ahead is a turn-based survival game set in a post-apocalypt
 
 #### Arch Linux
 
-Ncurses and tiles versions are available in the [community repo](https://www.archlinux.org/packages/?q=cataclysm-dda).
+Ncurses and tiles versions are available in the [offical repos](https://www.archlinux.org/packages/?q=cataclysm-dda).
```

Comment:
In this context the dda project is official and arch is not part of the dda project.
```suggestion
Ncurses and tiles versions are available in the [community repos](https://www.archlinux.org/packages/?q=cataclysm-dda).
```